libsplinter/src/admin/service/shared.rs

```diff
@@ -506,6 +561,18 @@ impl AdminServiceShared {
                                 info!("committed vote for circuit proposal {}", circuit_id);
                                 Ok(())
                             }
+                            CircuitManagementPayload_Action::CIRCUIT_DISBAND_REQUEST => {
+                                // notify registered application authorization handlers of the
+                                // committed disband circuit proposal
+                                let event = messages::AdminServiceEvent::ProposalSubmitted(
+                                    messages::CircuitProposal::from_proto(circuit_proposal.clone())
+                                        .map_err(AdminSharedError::InvalidMessageFormat)?,
+                                );
+                                self.send_event(&mgmt_type, event);
+
+                                info!("committed changes to disband circuit {}", circuit_id);
```

Comment:
This should say `info!("committed changes for new circuit proposal to disband circuit {}", circuit_id);` and the create log above should say to `proposal to create circuit {}`